### ISSUE TITLE
Proper Login Screen

### ISSUE THREAD

**Issue by dergigi** [joinmarket-webui/joinmarket-webui#120]
The HTTP Basic Authentication added in #75 was just a temporary solution to get the first version out the door. 

We should probably implement a proper login screen just like RTL and others have it.

**Comment by editwentyone** [comment]
if we implement a proper login screen, do we then need individual passwords for each wallet inside?

**Comment by dnlggr** [comment]
Nope, this would be independent from wallet passwords. UX wise it would stay the same as it is with the current [Basic Authentication](https://en.wikipedia.org/wiki/Basic_access_authentication) setup.

**Comment by editwentyone** [comment]
then why bother? I mean you can't access the wallets tu unlock them. the only option you have is to create a new wallet (until now)

why bother the normal home user to locking kind of twice (even 3 times, if you go through umbrel login » apps)

**Comment by dnlggr** [comment]
Because some parts of the API (e.g. listing wallets, creating wallets) will be accessible for anyone on the internet otherwise. While, as you correctly said, funds won't be in danger since wallets are locked on Joinmarket level, imagine people harassing you by creating thousands of wallets on your node or inferring things about the way you use Joinmarket by looking at what/how many wallets you have. That's something we don't want.

We already have this flow in place. It uses HTTP Basic Authentication to implement the vertical line from the diagram. Basic Auth looks a bit old and dusted and there's no way for us to style it which is why we want to move towards an in-app login screen similar to what Thunderhub, RTL, etc. have.

**Comment by editwentyone** [comment]
I added a simple login screen and also a logout possibility on the settings screen

https://www.figma.com/file/kfejZJFlwBywvLEnPEmJo1/?node-id=3994%3A73345

**Comment by theborakompanioni** [comment]
v2 has a proper login screen.
